justfile

```diff
@@ -77,7 +77,7 @@ check-fmt:
     cargo fmt --all -- --check
 
 clippy: (_target-installed target)
-    cargo clippy {{ _target-option }} --all-targets --workspace -- -D warnings
```

Comment:
What's the additional compilation time due to using `hack --each-feature` on CI and locally?

Reply:
`hack clippy` takes 18 minutes and `clippy` takes 4 minutes on my computer. If it's too long, maybe we can run `hack --each-feature` periodically ( per day or per week)?

Reply:
I have add an daily workflow, ptal